config.json

```diff
@@ -11,404 +11,404 @@
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "queen-attack",
-      "difficulty": 1,
+      "slug": "house",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "binary-search-tree",
-      "difficulty": 1,
+      "slug": "pascals-triangle",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "difference-of-squares",
-      "difficulty": 1,
+      "slug": "pythagorean-triplet",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "largest-series-product",
-      "difficulty": 1,
+      "slug": "saddle-points",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "luhn",
-      "difficulty": 1,
+      "slug": "series",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "clock",
-      "difficulty": 1,
+      "slug": "sieve",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "matrix",
-      "difficulty": 1,
+      "slug": "atbash-cipher",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "house",
-      "difficulty": 1,
+      "slug": "crypto-square",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "zebra-puzzle",
-      "difficulty": 1,
+      "slug": "nth-prime",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "minesweeper",
-      "difficulty": 1,
+      "slug": "binary-search-tree",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "ocr-numbers",
-      "difficulty": 1,
+      "slug": "luhn",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "wordy",
-      "difficulty": 1,
+      "slug": "ocr-numbers",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
       "slug": "food-chain",
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "linked-list",
-      "difficulty": 1,
+      "slug": "custom-set",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "custom-set",
-      "difficulty": 1,
+      "slug": "simple-cipher",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "nth-prime",
-      "difficulty": 1,
+      "slug": "parallel-letter-frequency",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
       "slug": "palindrome-products",
-      "difficulty": 1,
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pascals-triangle",
-      "difficulty": 1,
+      "slug": "pig-latin",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pig-latin",
-      "difficulty": 1,
+      "slug": "robot-name",
+      "difficulty": 6,
       "topics": [
       ]
     },
     {
-      "slug": "pythagorean-triplet",
-      "difficulty": 1,
+      "slug": "bank-account",
```

Comment:
bank--account and robot-name (both IO monad) next to each other - is that OK, or should they be separate? is it good to have two consecutive to reinforce the concept, or apart so that people get a break?